software/nosql/src/main/java/brooklyn/entity/nosql/redis/RedisClusterImpl.java

```diff
@@ -80,4 +84,26 @@ protected boolean calculateServiceUp() {
         return up;
     }
 
+    @Override
+    public void init() {
+        super.init();
+
+        subscribe(master, RedisStore.HOSTNAME, new MasterAttributePropagator());
```

Comment:
You could instead use:

    addEnricher(Enrichers.builder()
        .propagating(RedisStore.HOSTNAME, RedisStore.ADDRESS, RedisStore.SUBNET_HOSTNAME, RedisStore.SUBNET_ADDRESS, RedisStore.REDIS_PORT)
        .from(master)
        .build());